### ISSUE TITLE
Error when testing connection using tor 

### ISSUE THREAD

**Issue by bitstrologer** [sparrowwallet/sparrow#62]
This happens while attempting to use the blockstream tor address shown in [the guide](https://sparrowwallet.com/docs/quick-start.html#sparrow-server-setup).

```
2021-02-25 11:16:46,561 ERROR [JavaFX Application Thread] c.s.s.p.ServerPreferencesController [null:-1] Connection error
java.io.IOException: Cannot run program "uname": error=13, Permission denied
	at java.base/java.lang.ProcessBuilder.start(Unknown Source)
	at java.base/java.lang.ProcessBuilder.start(Unknown Source)
	at java.base/java.lang.Runtime.exec(Unknown Source)
	at java.base/java.lang.Runtime.exec(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/org.berndpruenster.netlayer.tor.OsType$Companion.getLinuxType(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/org.berndpruenster.netlayer.tor.OsType$Companion.access$getLinuxType(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/org.berndpruenster.netlayer.tor.OsType$Companion$current$2.invoke(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/org.berndpruenster.netlayer.tor.OsType$Companion$current$2.invoke(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/kotlin.SynchronizedLazyImpl.getValue(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/org.berndpruenster.netlayer.tor.OsType$Companion.getCurrent(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/org.berndpruenster.netlayer.tor.NativeContext$rcPath$2.invoke(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/org.berndpruenster.netlayer.tor.NativeContext$rcPath$2.invoke(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/kotlin.SynchronizedLazyImpl.getValue(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/org.berndpruenster.netlayer.tor.NativeContext.getRcPath(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/org.berndpruenster.netlayer.tor.NativeContext.<init>(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/org.berndpruenster.netlayer.tor.NativeTor.<init>(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/org.berndpruenster.netlayer.tor.NativeTor.<init>(Unknown Source)
	at com.sparrowwallet.merged.module@1.2.0/org.berndpruenster.netlayer.tor.NativeTor.<init>(Unknown Source)
	at com.sparrowwallet.sparrow@1.2.0/com.sparrowwallet.sparrow.net.TorService$1.call(Unknown Source)
	at com.sparrowwallet.sparrow@1.2.0/com.sparrowwallet.sparrow.net.TorService$1.call(Unknown Source)
	at javafx.graphics/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at javafx.graphics/javafx.concurrent.Service.lambda$executeTask$6(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at javafx.graphics/javafx.concurrent.Service.lambda$executeTask$7(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: error=13, Permission denied
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(Unknown Source)
	at java.base/java.lang.ProcessImpl.start(Unknown Source)
```

I'm using the linux standalone version.

```
[bitstrologer:~] $ uname -a | xargs echo | cut -d " " -f 1,3-15 
Linux 5.10.13-arch1-1 #1 SMP PREEMPT Wed, 03 Feb 2021 23:44:07 +0000 x86_64 GNU/Linux
```

Steps to Reproduce:

1. Go to server setup section
2. Attempt to use tor address in setup guide to connect to
3. Note error

**Comment by bitstrologer** [comment]
Sorry for the delay in my response:

```
[bitstrologer:~] $ which uname | xargs ls -l
-rwxr-xr-x 1 root root 39080 Mar  6  2020 /usr/bin/uname
```

```
[bitstrologer:~] 130 $ uname -a
Linux [hostname redacted] 5.11.8-arch1-1 #1 SMP PREEMPT Sun, 21 Mar 2021 01:55:51 +0000 x86_64 GNU/Linux
```

**Comment by craigraw** [comment]
There was an issue in the Linux standalone version which was only fixed in 1.3.1, and might have caused this. Can you try again with the most recent version? Thanks

**Comment by bitstrologer** [comment]
I no longer get this when testing the connection with the latest version, closing.
